pageserver/src/tenant/config.rs

```diff
@@ -385,6 +388,10 @@ pub struct TenantConfOpt {
     #[serde(skip_serializing_if = "Option::is_none")]
     #[serde(default)]
     pub gc_feedback: Option<bool>,
+
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
Making this property global-only is probably fine -- then we can avoid the complexity of carrying it as a per-tenant config as well.